### ISSUE TITLE
Git sync throws ent error on renaming

### ISSUE THREAD

**Issue by jensg-st** [direktiv/direktiv#643]
**Describe the bug**

I renamed a few files in https://github.com/direktiv/apps-svc. The follwoing rename actions were applied:

```
 rename generate-tags.yaml => generate-tags._yaml (100%)
 rename get-html.yaml => get-html._yaml (100%)
 rename get-swagger.yaml => get-swagger._yaml (100%)
 rename list.yaml => list._yaml (100%)
 rename load-spec.yaml => load-spec_.yaml (100%)
 rename load-specs.yaml => load-specs._yaml (100%)
 rename md.yaml => md._yaml (100%)
 rename pq.yaml => pq._yaml (100%)
 rename spec.yaml => spec._yaml (100%)
 rename tags.yaml => tags._yaml (100%)
 rename uri-key.yaml => uri-key._yaml (100%)
 rename uri-tag-key.yaml => uri-tag-key._yaml (100%)
```

The sync fails with :+1: 

```
Mirror activity 'sync' failed: ent: validator failed for field "Inode.name": value does not match validation
```

The last log lines: 

```
{"level":"info","timestamp":"2022-07-26T08:27:34Z","caller":"flow/logs.go:126","msg":"Deleted workflow '/get-swagger'.","component":"flow","build":"10e25bf","trace":"00000000000000000000000000000000","namespace":"svc","namespace-id":"8fb6dc8f-1b1d-470b-a5fa-d44b1ed75f80"}
{"level":"debug","timestamp":"2022-07-26T08:27:34Z","caller":"flow/pubsub.go:517","msg":"PS Notify Inode: c4f1d45f-8ca0-43d3-a968-26627cdb7090","component":"flow","build":"10e25bf"}
{"level":"info","timestamp":"2022-07-26T08:27:34Z","caller":"flow/logs.go:126","msg":"Deleted workflow '/uri-tag-key'.","component":"flow","build":"10e25bf","trace":"00000000000000000000000000000000","namespace":"svc","namespace-id":"8fb6dc8f-1b1d-470b-a5fa-d44b1ed75f80"}
{"level":"debug","timestamp":"2022-07-26T08:27:34Z","caller":"flow/pubsub.go:517","msg":"PS Notify Inode: c4f1d45f-8ca0-43d3-a968-26627cdb7090","component":"flow","build":"10e25bf"}
{"level":"info","timestamp":"2022-07-26T08:27:34Z","caller":"flow/logs.go:271","msg":"Mirror activity 'sync' failed: ent: validator failed for field \"Inode.name\": value does not match validation","component":"flow","build":"10e25bf","trace":"00000000000000000000000000000000","namespace":"svc","namespace-id":"8fb6dc8f-1b1d-470b-a5fa-d44b1ed75f80","mirror-id":"13340a02-1184-4ad5-b8c7-9413ae545d96"}

```

It seems to be obvious that it is cause by the renaming `rename load-spec.yaml => load-spec_.yaml `. The solution is to improve the logs? Because this was a rename to an invalid workflow name.

**Comment by alankm** [comment]
So there are two reasons this failed:

1. Underscores aren't allowed in inode names on Direktiv. At least, not as the final character anyway. In just one of those renames, you had a typo that put the underscore on the left of the dot. `rename load-spec.yaml => load-spec_.yaml (100%)`

2. The logic syncer logic isn't gracefully handling this. 

I don't think we need to change what names are permissable. And I don't think the syncer logic should try to protect you from this. I think in this case it should just delete every one of these workflows without attempting to rename them. Do you agree?

**Comment by jensg-st** [comment]
I know that the issue was with the naming and the behaviour is actually ok. I think the cryptic error  message was/is the problem.

**Comment by alankm** [comment]
I think this was resolved with PR #642
